.github/workflows/release.yml

```diff
@@ -58,3 +58,5 @@ jobs:
             ghcr.io/${{ env.GHCR_REPOSITORY }}:${{ steps.refs.outputs.SOURCE_TAG }}
             ghcr.io/${{ env.GHCR_REPOSITORY }}:latest
           platforms: linux/amd64,linux/arm64,linux/arm/v7
+          build-args: |
+            "CGO_ENABLED=${{ env.CGO_ENABLED }}"
```

Comment:
Do you even need the quotation in here?

Reply:
updated the PR to remove the erroneous quotes. good shout!